src/lms-ds-loader/lms_ds_loader/arguments.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+
+class DbConnection:
+    @staticmethod
+    def build_for_mssql_with_integrated_security(server, port, db_name):
+        assert server is not None
+        assert server.strip() != ""
+
+        assert db_name is not None
+        assert db_name.strip() != ""
+
+        if not port:
+            port = 1433
+        if type(port) == str and port.strip() == "":
+            port = 1433
+
+        return f"mssql+pyodbc://{server},{port}/{db_name}?driver=SQL Server?Trusted_Connection=yes"
+
+    @staticmethod
+    def build_for_mssql(server, port, db_name, username, password):
+        assert server is not None
+        assert server.strip() != ""
+
+        assert db_name is not None
+        assert db_name.strip() != ""
+
+        if not port:
+            port = 1433
+        if type(port) == str and port.strip() == "":
+            port = 1433
+
+        return f"mssql+pyodbc://{username}:{password}@{server},{port}/{db_name}?driver=SQL Server"
+
+
+class Arguments:

```

Comment:
For these kind of simple classes, check out [dataclass](https://docs.python.org/3/library/dataclasses.html), new in Python 3.7. It's an easier way to define them and provides conversions to Dicts and Tuples if you so choose.

Reply:
Will take a look.

Reply:
I've deferred looking into that, can refactor in the next ticket if desired.

Reply:
I like what I see with the `dataclass`. But there is something confusing about it. @lalrg pointed out that `my_var` in the sample below is a _class variable_, not an _instance variable_ as I had assumed ([confirmation](https://docs.python.org/3/tutorial/classes.html#class-and-instance-variables)).

```python
class MyClass:
    my_var: str = None
```

Now if we write that as a data class, `my_var` is now an instance variable. So if you don't read the class carefully - and miss that it is a `dataclass`, then this can be a little surprising. FYI, [here's a test](https://ideone.com/dnwfo5) that proves it is an instance var, not class var.

```python
@dataclass
class MyClass:
    my_var: str
```

Side note: there is a really import point on mutable instance variables (list, dict, tuple) in the [dataclass](https://docs.python.org/3/library/dataclasses.html#mutable-default-values).